### PR TITLE
Regenerate bindings for DeviceGetProcessUtilization change

### PR DESCRIPTION
**Pull request by elezar**
This adds the regenerated bindings from #12 

Signed-off-by: Evan Lezar <elezar@nvidia.com>

### CODE REVIEW

File: pkg/nvml/nvml.go [hunk 1]
```diff
@@ -25,2283 +25,2522 @@ package nvml
...
 
 // nvmlInit_v2 function as declared in nvml/nvml.h:1766
-func nvmlInit_v2() Return {
-	__ret := C.nvmlInit_v2()
-	__v := (Return)(__ret)
-	return __v
+func nvmlInit_v2() Return{
+__ret := C.nvmlInit_v2()
+__v := (Return)(__ret)
+return __v
```
Comment (klueska) [inline pkg/nvml/nvml.go:33]: What are all these changes in this file from?

File: pkg/nvml/types_gen.go [hunk 2]
```diff
@@ -1,345 +0,0 @@
-// Code generated by cmd/cgo -godefs; DO NOT EDIT.
```
Comment (klueska) [inline pkg/nvml/types_gen.go:1]: This file should definitely not be deleted....

Comment (elezar) [reply pkg/nvml/types_gen.go:1]: Seems my docker vm has a full disk.

File: pkg/nvml/nvml.go (hunk 1 above)
Comment (elezar) [reply pkg/nvml/nvml.go]: Was also due to the insufficient disk space. I missed the warnings / errors.

**Review by klueska — APPROVED** [review]

**Pull request merged**